modules/apps/asset/asset-entry-query-processor-custom-user-attributes/src/main/java/com/liferay/asset/entry/query/processor/custom/user/attributes/internal/CustomUserAttributesAssetEntryQueryProcessor.java

```diff
@@ -116,8 +116,7 @@ private void _addUserAttributes(
 						assetCategory.getVocabularyId());
 
 				if (Objects.equals(
-						customUserAttributeName,
-						assetVocabulary.getTitleCurrentValue())) {
+						customUserAttributeName, assetVocabulary.getName())) {
```

Comment:
Hi @TaiPhamLR 

With this change, the https://issues.liferay.com/browse/LPS-130127 is not working and will result on a regression.

Can you take a look so, both tickets work properly?

Thanks!

Reply:
Hi @AliciaGarciaGarcia, thanks for your feedback. I tested the change for LPS-130127 and couldn't reproduce the issue. So please just ignore it. Besides, I updated the code for some missing some case. Thanks!